### PR TITLE
translated home page to hindi

### PR DESCRIPTION
**Pull request by saranshkataria**
translated rest of the home page to hindi

### CODE REVIEW

File: content/home/examples/a-simple-component.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+title: एक साधारण कौम्पोनॅन्ट
 order: 0
 domid: hello-example
 ---
 
-React components implement a `render()` method that takes input data and returns what to display. This example uses an XML-like syntax called JSX. Input data that is passed into the component can be accessed by `render()` via `this.props`.
+React कौम्पोनॅन्ट एक `render()` मेथड लागू करते हैं जो इनपुट डेटा लेता है और यह दिखाता है कि क्या प्रदर्शित करना है। यह उदाहरण XML- जैसे सिंटैक्स का उपयोग करता है जिसे JSX कहा जाता है। इनपुट डेटा जिसे कौम्पोनॅन्ट में पास किया जाता है, उसे `this.props` के माध्यम से `render()` द्वारा एक्सेस किया जा सकता है।
 
-**JSX is optional and not required to use React.** Try the [Babel REPL](babel://es5-syntax-example) to see the raw JavaScript code produced by the JSX compilation step.
+**JSX इच्छाधीन है और React का उपयोग करने की आवश्यकता नहीं है।** JSX द्वारा निर्मित जावास्क्रिप्ट कोड को देखने के लिए [Babel REPL](babel://es5-syntax-example) का उपयोग करें।
```
Comment (arshadkazmi42) [inline content/home/examples/a-simple-component.md]: `**JSX इच्छाधीन है और React का उपयोग करने की आवश्यकता नहीं है।**`
I think this should be something like this
`**JSX इच्छाधीन है और React का उपयोग करने के लिए इसकी आवश्यकता नहीं है।**`

File: content/home/examples/an-application.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-title: An Application
+title: एक ऐप्लिकेशन
 order: 2
 domid: todos-example
 ---
 
-Using `props` and `state`, we can put together a small Todo application. This example uses `state` to track the current list of items as well as the text that the user has entered. Although event handlers appear to be rendered inline, they will be collected and implemented using event delegation.
+`props` और `state` का उपयोग करके, हम एक छोटे से टोडो ऐप्लिकेशन बना सक्ते हैं। यह उदाहरण आइटम की वर्तमान सूची के साथ-साथ उपयोगकर्ता द्वारा दर्ज किए गए टेक्स्ट को ट्रैक करने के लिए `state` का उपयोग करता है। हालाँकि, ईवेंट हैंडलर को इनलाइन प्रदान किया जाना चाहिए, लेकिन उन्हें इवेंट डेलिगेशन का उपयोग करके एकत्र किया गया है।
```
Comment (arshadkazmi42) [inline content/home/examples/an-application.md]: ` छोटे से ` this should be `छोटी सी`

File: src/pages/index.js [hunk 3]
```diff
@@ -51,7 +51,7 @@ class Home extends Component {
     return (
       <Layout location={location}>
         <TitleAndMetaTags
-          title="React &ndash; A JavaScript library for building user interfaces"
+          title="React &ndash; User interfaces के निर्माण के लिए एक जावास्क्रिप्ट लाइब्रेरी"
```
Comment (arshadkazmi42) [inline src/pages/index.js]: Any reason you have not translated `user interfaces`?

Comment (saranshkataria) [reply src/pages/index.js]: We were not translating ui, so left that as well, or should I?

Comment (arshadkazmi42) [reply src/pages/index.js]: I feel that is only for `UI` since its a shorthand

Comment (saranshkataria) [reply src/pages/index.js]: what will be the translation of it?

Comment (arshadkazmi42) [reply src/pages/index.js]: `युज़र इंटरफ़ेस`
How about this.
I checked its hindi meaning. its more complex. How about keeping it like this?

Comment (saranshkataria) [reply src/pages/index.js]: interfaces, plural version is the problem 😞

Comment (arshadkazmi42) [reply src/pages/index.js]: `इंटरफेसेस`
How about this?

Comment (saranshkataria) [reply src/pages/index.js]: looks confusing to me

Comment (arshadkazmi42) [reply src/pages/index.js]: Then let's keep it `User Interfaces`.
We will come back to this, if it needs any changes

Comment (saranshkataria) [reply src/pages/index.js]: I changed it to युज़र इंटरफ़ेसिज़ and committed it along with the other changes you asked for

**Pull request merged**